eng/Versions.props

```diff
@@ -13,4 +13,7 @@
     <MicrosoftTestingInternalFrameworkVersion>1.5.0-preview.24560.1</MicrosoftTestingInternalFrameworkVersion>
     <MSTestEngineVersion>1.0.0-alpha.24473.2</MSTestEngineVersion>
   </PropertyGroup>
+  <PropertyGroup Label="MSTest prod dependencies">
+    <SystemThreadingTasksExtensionsVersion>4.5.4</SystemThreadingTasksExtensionsVersion>
```

Comment:
Let's have the version in the `Directory.Packages.props` file. I am trying to have only darc/maestro updated versions here.